codespell_lib/data/dictionary.txt

```diff
@@ -36398,6 +36681,8 @@ tiemstamp->timestamp
 tiemstamped->timestamped
 tiemstamps->timestamps
 tieth->tithe
+tification->notification
```

Comment:
This is strange, could also be `ratification`, `certification`, `fortification`, `justification`, `rectification`, `mortification`. Why this specific fix?

Ah wait, after looking it up in GitHub, it looks like some programmers might write “notification” as “no-tification”. For example:
https://github.com/mavenjan/BaseApp-master/blob/a5131f991afabc0c3a9058b258568245576179a6/mjlibrary/src/main/java/com/cauc/mavenj/utils/NotificationUtil.java#L22

What do you think?

Reply:
> This is strange, could also be `identification`, `ratification`, `certification`, `fortification`, `justification`, `rectification`, `mortification`. Why this specific fix?

Hmm, you're right. 

> What do you think?

How about just adding in all the above alternatives? "tification" can never be right, in any case.

Reply:
I don't know. This looks very specific. `tification` might appear when the complete word is split at the end of the line:
```
... no-
tification ...
```

Reply:
There are a 20k hits here: https://github.com/search?q=tification&type=code

> `tification` might appear when the complete word is split at the end of the line:

Or in [quantification](https://github.com/sungunrun/index0926/blob/ad29eacd0a47fe312fecb055e12088dddde952ad/static/page_contents/test_196.html#L43), [justification](https://github.com/Deuy76/R_Textming_Tini/blob/c79aabf7dad6329d829c996792873ea181fa60c2/data/20news-bydate/20news-bydate-train/comp.os.ms-windows.misc/9996#L69), etc.

I'm honestly not sure. The options I see are either adding all variants, or deleting the entry.

Reply:
To keep it simple let's delete it. It's probably not sustainable to add truncated/split words

Reply:
```suggestion
```

Reply:
> To keep it simple let's delete it. It's probably not sustainable to add truncated/split words

Yup, makes sense.